R/backport_linter.R

```diff
@@ -74,7 +74,7 @@ backport_linter <- function(r_version = getRversion(), except = character()) {
     bad_versions <- unname(backport_index[all_names])
 
     lint_message <- sprintf(
-      "%s (R %s) is not available for dependency R >= %s.",
+      "%s (R %s) is not available for all versions of R >= %s.",
```

Comment:
How about keeping it succinct?

```suggestion
      "%s (R %s) is not available for R versions >= %s.",
```

Reply:
I believe "all" is key. Its absence is what confused me. Because I was thinking 3.2.0 >= 3.0.0 so it seemed odd to read

> trimws (R 3.2.0) is not available for all versions of R >= 3.0.0.

Reply:
What about 

> "%s (R %s) is not always available for requested dependency (R >= %s)."

then?

But maybe I'm getting stuck in a "me" thing. If you both believe the current phrasing is good enough, I'm happy with keeping it and closing this PR.

Reply:
no I think there's definitely room for improvement here. I only see "always" in your version as different from my second suggestion, how should I interpret that?

Reply:
> I only see "always" in your version as different from my second suggestion, how should I interpret that?

I would put always in both suggestions. I might prefer slightly the second form because it immediately creates a connection with the format used in `DESCRIPTION` but I don't feel very strongly at all.